README.md

```diff
@@ -1 +1,100 @@
-# presto-js-client
\ No newline at end of file
+# Presto JS Client
+
+This is a Presto JavaScript client that connects to Presto via Presto's REST API to run queries.
+
+## Installation
+
+```sh
+npm install @prestodb/presto-js-client
+```
+
+## Usage
+
+Import the PrestoClient class from `@prestodb/presto-js-client`.
+
+Create a new instance by passing the connection parameters.
+
+### Example
+
+```typescript
+import PrestoClient from '@prestodb/presto-js-client'
+
+...
+
+const client = PrestoClient({
+  catalog: 'tpcs',
+  host: 'localhost',
+  interval: 0,
+  port: 8080,
+  schema: 'tiny',
+  timezone: 'America/Costa_Rica',
+  user: 'root'
+})
+```
+
+### Parameters
+
+The Presto client can be configured with the following parameters:
+
+- `host`: The hostname or IP address of the Presto coordinator. (Default: `http://localhost`)
```

Comment:
We need to update this list with the latest changes in `PrestoClientConfig`

Reply:
done